### PR TITLE
Solve pylint invalid-name

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

**Comment by berland** [comment]
Left as three commits for the sake of review.

The three remaining exceptions in .pylintrc are not something that should be fixed, thus closing the pylinting issue.

File: .pylintrc [hunk 1]
```diff
@@ -135,7 +132,7 @@ include-naming-hint=no
 inlinevar-rgx=[A-Za-z_][A-Za-z0-9_]*$
 
 # Regular expression matching correct method names
-method-rgx=(([a-z][a-z0-9_]{2,30})|(_[a-z0-9_]*))$
+method-rgx=(([a-z][a-z0-9_]{2,50})|(_[a-z0-9_]*))$
```
Comment (hnformentin) [inline .pylintrc:135]: Why need to increase from 30 to 50?
`{2,50}` means match at least two times but no more than 50 times, is it correct?
I see that [below](https://github.com/equinor/semeio/pull/518/files#diff-ba293004ae8f18975051e825bd8d5c70137a8b5d42d16e0f7c9354aa634c446cL156) we keep 30, so wondered the reason behind this change.

File: semeio/workflows/localisation/local_script_lib.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# pxylint: disable=attribute-defined-outside-init
+# pylint: disable=attribute-defined-outside-init,invalid-name
```
Comment (hnformentin) [inline semeio/workflows/localisation/local_script_lib.py:1]: Was `pxylint` only misspelled?
Why include `invalid-name`?

File: semeio/workflows/localisation/localisation_config.py [hunk 3]
```diff
@@ -216,10 +216,9 @@ class ScalingForSegmentsConfig(BaseModel):
     ] = [0, 0]
 
     @validator("scalingfactors")
-    def check_length_consistency(cls, v, values):
+    def check_length_consistency(cls, scalingfactors, values):
```
Comment (hnformentin) [inline semeio/workflows/localisation/localisation_config.py:219]: Any reason not to be `scaling_factors`?

File: tests/hook_implementations/test_hook_implementations.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 
 def test_hook_implementations():
-    pm = ErtPluginManager(
+    pluginmanager = ErtPluginManager(
```
Comment (hnformentin) [inline tests/hook_implementations/test_hook_implementations.py:15]: plugin_manager?

File: .pylintrc (hunk 1 above)
Comment (berland) [reply .pylintrc:135]: There are some test function names that are longer than 30 characters. I think those test function names are legitimate and not something I want to disallow, so therefore I propose allowing up to 50 at least. One could think of having different requirements for test function names and production function names, but I think that just adds unnecessary complications to the CI.

File: tests/hook_implementations/test_hook_implementations.py (hunk 4 above)
Comment (berland) [reply tests/hook_implementations/test_hook_implementations.py]: Fixed. `plugin_manager` is in line with the spelling inside the module `ert.shared.plugins`. Hopefully that will not lead to more confusion.

**Review by hnformentin — APPROVED** [review]
Thanks for the replies 🚀

**Pull request merged**